### ISSUE TITLE
Upgrading bubbletea v0.25.0 → bubbletea v0.26.6 breaks table rendering

### ISSUE THREAD

**Issue by dnalor** [Evertras/bubble-table#178]
![ttui-rend](https://github.com/user-attachments/assets/04947490-7238-4fd8-9de3-ef622b1ca725)

(if you scroll over the rows, they are redrawn correctly)

Most likely a problem in my application, but afaik I'm not anything unusual with the table, so any suggestion will be appreciated.

Code is at [codeberg](https://codeberg.org/dnalor/tuiarchiver/) if you want to have a look (internal/tui/tui.go)

**Comment by Evertras** [comment]
When I git clone your repository and update bubbletea v0.26.6, I don't seem to see this behavior out of the gate.  However, I do see all kinds of artifacts if I resize the terminal, regardless of whether I update or not.  I also see this with the flex example locally here, though, so maybe I need to investigate this more to see if there's some redrawing thing we're both missing.

**Comment by dnalor** [comment]
Weird, now I also cannot reproduce the behaviour shown in the screenshot above. Maybe some othere package was updated in the meantime? 

Currently, when resizing, if the table isn't smaller than the terminal, everything works fine.

However, if there's a vertical resize beyond table size:
- urxvt: broken:
![v25-vr-urxvt sm](https://github.com/user-attachments/assets/98205765-a0e2-41ff-9d1f-ed87a0440526)

- kitty: initially ok, but if then resized horizontally, it also breaks:
![v25-vr-hr-kitty sm](https://github.com/user-attachments/assets/7a01af33-570e-4d5e-b197-30d65e9d1ab8)

This happens for all combinations of bubbletea (v0.25.0, v0.26.6) and bubble-table (v0.15.6, v0.16.1) I tested, so it seems it's not a new issue.
